docs/Common/Structs/index.md

```diff
@@ -752,6 +752,10 @@
 |:---|:---|:--------|:---------|:----------|
 |type|[Common.FuelType](../enums/#fueltype)|false|||
 |range|Float|false|minvalue: 0<br>maxvalue: 10000|The estimate range in KM the vehicle can travel based on fuel level and consumption|
+|level|Float|false|minvalue: 0<br>maxvalue: 10000|The relative remaining capacity of this fuel type (percentage)|
+|levelState|[Common.ComponentVolumeStatus](../enums/#componentvolumestatus)|false||The fuel level state|
+|capacity|Float|false|minvalue: 0<br>maxvalue: 10000|The absolute capacity of this fuel type|
```

Comment:
maxvalue: 1000000

Reply:
updated in 86be035deb4426cac61bd292dae37c3be8ae1f64